otel/internal/utils/sentryrequest.go

```diff
@@ -0,0 +1,40 @@
+//go:build go1.18
+
+package utils
+
+import (
+	"context"
+	"strings"
+
+	"github.com/getsentry/sentry-go"
+	"go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
+)
+
+func IsSentryRequestSpan(ctx context.Context, s trace.ReadOnlySpan) bool {
+	attributes := s.Attributes()
+
+	// TODO(michi): can we access the attribute directly?
```

Comment:
I haven't found a better way, so we might roll with this instead. Will remove the comments if you're 👍

Reply:
Let's leave it like this for now, we can later implement something similar to what they do in sentry exporter: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/sentryexporter/sentry_exporter.go#L376-L394